### PR TITLE
add pyQParamWidget package

### PR DESCRIPTION
**Pull request by prjemian**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by prjemian** [comment]
I am willing to be a maintainer of this recipe.

**Comment by prjemian** [comment]
`meta.yaml` was built with `grayskull`

**Comment by prjemian** [comment]
Not understanding why [this happened here](https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/905502/logs/18):

```
2024-03-28T22:11:59.4240137Z     from PyQt5 import QtWidgets
2024-03-28T22:11:59.4240433Z ImportError: libGL.so.1: cannot open shared object file: No such file or directory
```

Amongst the package installs:

```
2024-03-28T22:11:34.1121858Z     libglib:                   2.80.0-hf2295e7_1         conda-forge
```

Was the package installed into the wrong place or what other problem?  Looks like an upstream problem.

**Comment by prjemian** [comment]
Could it be a line-length problem here?  Looks truncated:

```
2024-03-28T22:11:58.9590294Z export PREFIX=/home/conda/staged-recipes/build_artifacts/pyqparamwidget_1711663791066/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p
```

**Comment by prjemian** [comment]
.related:

- https://github.com/conda-forge/pygridgen-feedstock/issues/10
- https://conda-forge.org/docs/maintainer/knowledge_base/#libgl

**Comment by prjemian** [comment]
Or add  `libgl1-mesa-glx` to recipe:requirements:run

**Comment by prjemian** [comment]
Ok, comment out the import test worked.  But, kosher?  Restore and add requirement next.

**Comment by prjemian** [comment]
Failed result:

```
Encountered problems while solving:
  - nothing provides requested libgl1-mesa-glx
```

**Comment by prjemian** [comment]
Did not work.  Import failed as first reported.  Investigate the instructions to resolve with `libgl1-mesa-glx`.

**Comment by prjemian** [comment]
Compare with feedstock of another PyQt5 project: [PyDM](https://github.com/conda-forge/pydm-feedstock/blob/main/recipe/meta.yaml)

**Comment by prjemian** [comment]
@conda-forge-admin please rerender

**Comment by prjemian** [comment]
rerender does not work in this case.  At this stage, the recipe is not in a "feedstock".  Dig deeper into the [knowledge base](https://conda-forge.org/docs/maintainer/knowledge_base/#libgl) for more guidance.

**Comment by prjemian** [comment]
Hey!  That worked!

**Comment by prjemian** [comment]
Ready for review now.

File: recipes/pyQParamWidget/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+{% set version = "0.1.0" %}
+{% set sha256 = "51aa457b82f9bb038998788b9c3780c21a07c95f15d128f4b4ab0c841811e4b2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyQParamWidget-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (ocefpaf) [inline recipes/pyQParamWidget/meta.yaml:11]:
```suggestion

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyQParamWidget-{{ version }}.tar.gz
  sha256: 51aa457b82f9bb038998788b9c3780c21a07c95f15d128f4b4ab0c841811e4b2
```

**Comment by prjemian** [comment]
@ocefpaf - Your changes have been accepted.  Thanks for the feedback.  Ready for review.

**Pull request merged**